src/uwtools/resources/rocoto.jsonschema

```diff
@@ -130,6 +130,30 @@
           },
           "type": "object"
         },
+        "^sh(_.*)?$": {
+          "additionalProperties": false,
+          "properties": {
+            "attrs": {
+              "additionalProperties": false,
+              "properties": {
+                "runopt": {
+                  "type": "string"
+                },
+                "shell": {
+                  "type": "string"
+                }
+              },
+              "type": "object"
+            },
+            "command": {
+              "$ref": "#/$defs/compoundTimeString"
```

Comment:
We've used the more general `value:` key elsewhere. I'm unsure whether it's better to be consistent, or to be specific. In this case, "command" is specific.

Reply:
I like `command` better here.